src/backend/joanie/tests/core/test_api_order.py

```diff
@@ -1185,10 +1185,11 @@ def test_api_order_create_once(self):
         self.assertEqual(response.status_code, 400)
         self.assertEqual(
             response.json(),
-            (
-                f"Cannot create order related to the product {product.id} "
-                f"and course {course.code}"
-            ),
+            {
+                "__all__": [
+                    "Constraint “unique_owner_product_not_canceled” is violated."
```

Comment:
Maybe we should customize our constraint messages then? 
https://docs.djangoproject.com/en/4.2/ref/models/constraints/#violation-error-message